src/MangaKakalot/MangaKakalot.ts

```diff
@@ -0,0 +1,172 @@
+import {
+    BadgeColor,
+    ContentRating,
+    DUISection,
+    PagedResults,
+    PartialSourceManga,
+    SearchRequest,
+    SourceInfo,
+    SourceIntents,
+    Tag,
+    TagSection,
+} from '@paperback/types'
+
+import { decodeHTML } from 'entities'
+
+import {
+    MangaBox,
+    getExportVersion,
+} from '../MangaBox'
+
+import { chapterSettings } from '../MangaBoxSettings'
+
+import { URLBuilder } from '../MangaBoxHelpers'
+
+import {
+    isLastPage,
+    parseManga,
+} from '../MangaBoxParser'
+
+const SITE_DOMAIN = 'https://mangakakalot.com'
+
+export const MangaKakalotInfo: SourceInfo = {
+    version: getExportVersion('0.0.0'),
+    name: 'MangaKakalot',
+    icon: 'icon.png',
+    author: 'Batmeow',
+    authorWebsite: 'https://github.com/Batmeow',
+    description: `Extension that pulls manga from ${SITE_DOMAIN}.`,
+    contentRating: ContentRating.MATURE,
+    websiteBaseURL: SITE_DOMAIN,
+    sourceTags: [
+        {
+            text: 'Notifications',
+            type: BadgeColor.GREEN
+        }
+    ],
+    intents: SourceIntents.HOMEPAGE_SECTIONS | SourceIntents.MANGA_CHAPTERS
```

Comment:
Does Mangakakalot not have a second image server? Pretty sure it does?

Reply:
It does but the value it uses to determine which server to use is determined by a generated key. But then there are times when MangaKakalot redirects to the Manganato (chapmanganato) reader which in those cases the image server settings would match what Manganato & MangaBat use. I'll re-enable the setting for MangaKakalot so it can be used in those senarios.